Actions/CheckForUpdates/CheckForUpdates.ps1

```diff
@@ -109,21 +128,29 @@ if (-not $isDirectALGo) {
 # - All files in .github that ends with .copy.md
 # - All PowerShell scripts in .AL-Go folders (all projects)
 $checkfiles = @(
-    @{ 'dstPath' = Join-Path '.github' 'workflows'; 'srcPath' = Join-Path '.github' 'workflows'; 'pattern' = '*'; 'type' = 'workflow' },
-    @{ 'dstPath' = '.github'; 'srcPath' = '.github'; 'pattern' = '*.copy.md'; 'type' = 'releasenotes' },
-    @{ 'dstPath' = '.github'; 'srcPath' = '.github'; 'pattern' = 'AL-Go-Settings.json'; 'type' = 'settings' },
-    @{ 'dstPath' = '.github'; 'srcPath' = '.github'; 'pattern' = '*.settings.json'; 'type' = 'settings' }
+    @{ 'dstPath' = (Join-Path '.github' 'workflows'); 'newname' = ''; 'srcPath' = (Join-Path '.github' 'workflows'); 'pattern' = '*'; 'type' = 'workflow' },
```

Comment:
Minor: naming.

How about `dstName` to match the current patter ("src" for source and "dst" for destination)?

Reply:
agree